installation/installing-images/README.md

```diff
@@ -40,3 +40,12 @@ For more advanced control of this process, see our system-specific guides:
 - [Mac OS](mac.md)
 - [Windows](windows.md)
 - [Chrome OS](chromeos.md)
+
+
+## Boot your new OS
+
+You can now insert the SD card into the Raspberry Pi and power it up. 
+
+For the official Raspbian OS, if you need to enter it, the default user name is `pi`, with password `raspberry`. Remember the default keyboard layout is set to UK.
+
+You should change the default password straight away by running `passwd`, [as explained here](../../linux/usage/users.md)
```

Comment:
Isn't this already a suggested step in the "first run wizard"? ping @spl237

Reply:
Yes, it is part of the wizard; no need to do it from the terminal.

Reply:
What about Raspbian lite?

Reply:
Lite doesn't have the wizard, but it would seem to me that trying to use the same documentation for installing the desktop version as for lite is not the greatest idea...

Reply:
We've don't tend to distinguish, and TBH, there's very little documentation on the desktop in this repo at all.

Reply:
In that case, probably best to just recommend people change the password without being specific as to how...